dotnet/nuget/nuget-package.props

```diff
@@ -10,7 +10,7 @@
     <IsPackable>true</IsPackable>
 
     <!-- Package validation. Baseline Version should be lower or equal to current version. -->
-    <PackageValidationBaselineVersion>1.17.1</PackageValidationBaselineVersion>
+    <PackageValidationBaselineVersion>1.18.0-rc</PackageValidationBaselineVersion>
```

Comment:
We are likely doing a 1.17.2 next so we should remove this change until the OpenAI v2 stuff get's merged to main